esp/services/ws_workunits/ws_wudetails.cpp

```diff
@@ -415,8 +415,8 @@ void WUDetailsVisitor::resetScope()
     currentStatisticScopeType = SSTnone;
     extraStatisticsForCurScope = nullptr;
     extraAttributesForCurScope = nullptr;
-    EspWUResponseProperties.clear();
```

Comment:
IArrayOf::clear() seems guaranteed to leak - I wonder why the function exists, and whether it should... @ghalliday any views?

Reply:
I think the intent of the function is to free up the array, but not dispose of the array - because it is likely to be reused.  I suspect it should still call the destructors.  I'll see if anything actually calls it...

Reply:
Created HPCC-27770 - will push a fix once this is merged.
Should this target 8.6.x since it is a bug fix?

Reply:
Depends how severe the leak is I guess. But probably, yes

Reply:
@richardkchapman I've re-targeted to 8.6.x